core/command/const.go

```diff
@@ -15,7 +15,7 @@
  * @author: Spencer Bull, Dell
  * @version: 0.5.0
  *******************************************************************************/
-package main
+package command
 
 // ConfigurationStruct : Struct used to pase the JSON configuration file
 type ConfigurationStruct struct {
```

Comment:
No need to call it `ConfigurationStruct` - we see that it is `struct`. Go idiomatic way of naming variables suggest using very short and simple names.

Reply:
leaving for now as it is more cosmetic, but will address as I do more refactoring going forward.

Reply:
Agreed.